src/commands/local.ts

```diff
@@ -394,7 +395,24 @@ async function startProcesses(
     });
 
     const bundlersOutput = await Promise.all(bundlersOutputPromise);
-    await importServiceEnvVariables(projectConfiguration.name, projectConfiguration.region);
+
+    const authToken = await getAuthToken();
+    if (!authToken) {
+        process.env["LOGGED_IN_LOCAL"] = "false";
```

Comment:
Two observations:

- I would set the env variable only if the user is authenticated.
- I would name the variable something with the prefix `GNZ_` or `GENEZIO_` to avoid the chance of clashing with user's envs.